openassessment/xblock/static/sass/oa/elements/_staff.scss

```diff
@@ -91,14 +92,10 @@
   .openassessment_student_info_form {
     margin-bottom: ($baseline-v/2);
 
-    .list--actions__item {
```

Comment:
These elements are no longer wrapped in lists-- that was removed some time back, but the sass did not get updated. I wanted to change the "Submit" button above to $btn--primary, and thus found that this sass was not properly overriding.